locales/sk-SK.json

```diff
@@ -0,0 +1,1895 @@
+        ],
+        "2": [
+          "Odbočte vlevo na North Bond Street/US 1 Business/MD 924. Pokračujte na MD 924."
+        ],
+        "3": [
+          "Odbočte vpravo a zůstaňte na Sunstone Drive."
+        ]
+      }
+    },
+    "uturn": {
+      "phrases": {
+        "0": "Otočte sa <RELATIVE_DIRECTION>.",
+        "1": "Otočte sa <RELATIVE_DIRECTION> na <STREET_NAMES>.",
+        "2": "Otočte sa <RELATIVE_DIRECTION> a zostaňte na <STREET_NAMES>.",
+        "3": "Otočte sa <RELATIVE_DIRECTION> cez <CROSS_STREET_NAMES>.",
+        "4": "Otočte sa <RELATIVE_DIRECTION> cez <CROSS_STREET_NAMES> na <STREET_NAMES>.",
+        "5": "Otočte sa <RELATIVE_DIRECTION> cez <CROSS_STREET_NAMES> a zostaňte na <STREET_NAMES>."
+      },
+      "empty_street_name_labels": [
+        "chodník",
+        "cyklocestu",
+        "cestu pre horské bicykle"
+      ],
+      "relative_directions": [
+        "vľavo",
+        "vpravo"
+      ],
+      "example_phrases": {
+        "0": [
+          "Otočte se vlevo."
+        ],
+        "1": [
+          "Otočte se vpravo na Bunker Hill Road."
+        ],
+        "2": [
+          "Otočte se vlevo a zůstaňte na Bunker Hill Road."
+        ],
+        "3": [
+          "Otočte se vlevo přes Devonshire Road."
+        ],
+        "4": [
+          "Otočte se vlevo přes Devonshire Road na Jonestown Road/US 22."
+        ],
+        "5": [
+          "Otočte se vlevo přes Devonshire Road a zůstaňte na Jonestown Road/US 22."
+        ]
+      }
+    },
+    "uturn_verbal": {
+      "phrases": {
+        "0": "Otočte sa <RELATIVE_DIRECTION>.",
+        "1": "Otočte sa <RELATIVE_DIRECTION> na <STREET_NAMES>.",
+        "2": "Otočte sa <RELATIVE_DIRECTION> a zostaňte na <STREET_NAMES>.",
+        "3": "Otočte sa <RELATIVE_DIRECTION> cez <CROSS_STREET_NAMES>.",
+        "4": "Otočte sa <RELATIVE_DIRECTION> cez <CROSS_STREET_NAMES> na <STREET_NAMES>.",
+        "5": "Otočte sa <RELATIVE_DIRECTION> cez <CROSS_STREET_NAMES> a zostaňte na <STREET_NAMES>."
+      },
+      "empty_street_name_labels": [
+        "chodník",
+        "cyklocestu",
+        "cestu pre horské bicykle"
+      ],
+      "relative_directions": [
+        "vľavo",
+        "vpravo"
+      ],
+      "example_phrases": {
+        "0": [
+          "Otočte se vlevo."
+        ],
+        "1": [
+          "Otočte se vpravo na Bunker Hill Road."
+        ],
+        "2": [
+          "Otočte se vlevo a zůstaňte na Bunker Hill Road."
+        ],
+        "3": [
+          "Otočte se vlevo přes Devonshire Road."
+        ],
+        "4": [
+          "Otočte se vlevo přes Devonshire Road na Jonestown Road, U.S. 22."
+        ],
+        "5": [
+          "Otočte se vlevo přes Devonshire Road a zůstaňte na Jonestown Road, U.S. 22."
+        ]
+      }
+    },
+    "verbal_multi_cue": {
+      "phrases": {
+        "0": "<CURRENT_VERBAL_CUE> Potom <NEXT_VERBAL_CUE>",
+        "1": "<CURRENT_VERBAL_CUE> Then, in <LENGTH>, <NEXT_VERBAL_CUE>"
```

Comment:
Should this be `<CURRENT_VERBAL_CUE> Potom do <LENGTH>, <NEXT_VERBAL_CUE>` ?

Reply:
My bad, i will fix this